package.json

```diff
@@ -32,19 +32,25 @@
     "@babel/plugin-transform-runtime": "^7.21.0",
     "@babel/preset-env": "^7.20.2",
     "@babel/register": "^7.21.0",
+    "babel": "^6.23.0",
```

Comment:
- Remove babel in favor to babel/cli
- Remove also body-parser in favor to express.json()